debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleStreamingChangeEventSourceMetrics.java

```diff
@@ -472,7 +473,12 @@ else if (currentBatchSize > batchSizeMin) {
         }
 
         if (currentBatchSize == batchSizeMax) {
-            LOGGER.info("LogMiner is now using the maximum batch size {}. This could be indicative of large SCN gaps", currentBatchSize);
+            if (!lobEnabled) {
+                LOGGER.info("LogMiner is now using the maximum batch size {}. This could be indicative of large SCN gaps", currentBatchSize);
```

Comment:
```suggestion
                LOGGER.info("The connector is now using the maximum batch size {} when querying the LogMiner view. This could be indicative of large SCN gaps", currentBatchSize);
```